src/main/kotlin/kotlinx/kover/api/CoverageEngines.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017-2022 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:Suppress("RedundantVisibilityModifier")
+
+package kotlinx.kover.api
+
+import kotlinx.kover.api.KoverVersions.DEFAULT_INTELLIJ_VERSION
+import kotlinx.kover.api.KoverVersions.DEFAULT_JACOCO_VERSION
+import org.gradle.api.tasks.*
+
+public interface CoverageEngineVariant {
+    @get:Input
+    public val vendor: CoverageEngineVendor
+    @get:Input
+    public val version: String
+}
+
+public enum class CoverageEngineVendor {
+    INTELLIJ,
+    JACOCO
+}
+
+/**
+ * Coverage Engine by IntelliJ.
```

Comment:
If we decide to rename, we will need to do it in many places.
The current name was chosen because earlier in the IDEA settings it was signed as an IntelliJ Agent.

It seems to me that Kotlin Coverage Engine is not quite the correct name, because our agent is still working with Java code, and also this name seems to say what this engine is for, and not by whom it was made.

`JetBrains Coverage Engine` or `Kover Coverage Engine` (or `KoCoCo` :) )?